tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended"],
+  "jsRules": {},
+  "rules": {
+    "trailing-comma": [false],
```

Comment:
Just curious, is there a reason to disable the trailing comma? I tend to like it (but am OK either way).